### ISSUE TITLE
Remapping to multiple keys

### ISSUE THREAD

**Issue by jpoon** [VSCodeVim/Vim#725]
```
    "vim.otherModesKeyBindingsNonRecursive": [
    {
      "before": ["g", "h"],
      "after": ["ctrl+w", "h"]
      }]
```

The above to remap `gh` to `<C-w>h` does not work.  I see the keys being remapped from from `gh` to `<C-w>h`, however it looks like somewhere along the lines, `actionKeys` gets reset before it can handle both key commands.

Happens on latest (0.1.10).

**Comment by johnfn** [comment]
The following works for me: 

```
  "vim.insertModeKeyBindings": [
       {
           "before": ["j", "j"],
           "after": ["a", "a"]
       }
  ]
```

**Comment by rebornix** [comment]
Below works weird

```
  "vim.otherModesKeyBindings": [
    {
      "before": ["j", "j"],
      "after": ["d", "d"]
    }
  ]
```

It will move downward by one line and delete that line. Seems we are doing wrong with `otherModesKeyBindings`.

**Comment by xconverge** [comment]
Does it have to do with the keys repeating? The following doesn't work for me but jj does

"vim.insertModeKeyBindings": [
       {
           "before": [ "j", "k"],
           "after": ["<Esc>"]
       }
   ],

**Comment by johnfn** [comment]
I wonder if @xconverge fixed this with #732

**Comment by jpoon** [comment]
Ooohh, yes, I think that'll fix it.

**Comment by xconverge** [comment]
Woops

**Comment by johnfn** [comment]
This has been fixed on master and will be released on 0.4.7. The proper remapping looks like this:

```
        {
            "before": ["g", "h"],
            "after": ["<C-w>", "h"]
        },

        {
            "before": ["g", "l"],
            "after": ["<C-w>", "l"]
        }
```

**Comment by MarcosH911** [comment]
That remapping only works if you press the two keys in a shorter timespan than "vim.timeout". Is there a way to remap a combination of keys without having to press them in a certain amount of time?

For example, I would like to remap 'zz' to 'zt'
